src/custom_load_manager.cc

```diff
@@ -67,10 +67,10 @@ CustomLoadManager::CustomLoadManager(
     const std::unordered_map<std::string, cb::RequestParameter>&
         request_parameters)
     : RequestRateManager(
-          async, streaming, Distribution::CUSTOM, batch_size,
-          measurement_window_ms, max_trials, max_threads, num_of_sequences,
-          shared_memory_type, output_shm_size, serial_sequences, parser,
-          factory, request_parameters),
+          async, streaming, Distribution::CUSTOM, std::vector<float>{},
```

Comment:
Adding the new param early in the list has some down stream effects on the tests that make use of default values.
This should get added to the end of the param list.